hadoop-tools/hadoop-aws/src/test/java/org/apache/hadoop/fs/s3a/s3guard/ITestS3GuardConcurrentOps.java

```diff
@@ -55,6 +57,23 @@
   @Rule
   public final Timeout timeout = new Timeout(5 * 60 * 1000);
 
+  protected Configuration createConfiguration() {
+    Configuration conf =  super.createConfiguration();
+    //patch the read/write capacity
+    boolean scaleCapacityLimitEnabled = conf.getBoolean("fs.s3a.s3guard.ddb.table.scale.capacity.limit", true);
+    if(scaleCapacityLimitEnabled) {
+      LOG.info("Enable the capacity limit : {} -> {}, {} -> {}",
+        S3GUARD_DDB_TABLE_CAPACITY_READ_KEY, 1, S3GUARD_DDB_TABLE_CAPACITY_WRITE_KEY,1);
+      conf.set(S3GUARD_DDB_TABLE_CAPACITY_READ_KEY, "1");
+      conf.set(S3GUARD_DDB_TABLE_CAPACITY_WRITE_KEY, "1");
+    }
+    if(scaleCapacityLimitEnabled) {
```

Comment:
we don't need these -they don't show much.

What would be better would be the thread which actually creates a table asking for the bucket metadata and working out the capacity there. But I'm happy not to worry about that...All I'd prefer is to make sure that we don't run up any bills from test tables which weren't deleted